chainerx_cc/chainerx/routines/manipulation.cc

```diff
@@ -636,6 +636,86 @@ Array Swapaxes(const Array& a, int8_t axis1, int8_t axis2) {
     return out;
 }
 
+Array Repeat(const Array& a, int64_t repeats, nonstd::optional<int8_t> axis) {
+    if (repeats < 0) {
+        throw DimensionError("repeats must be larger than 0.");
+    }
+
+    int8_t targetAxis = 0;
+    Array targetArray;
```

Comment:
We have a naming convention for variable to use "snake_case". Could you change from camelCase to it?

Reply:
fixed